### ISSUE TITLE
[Wii U] RetroArch isn't seeing my USB Drive

### ISSUE THREAD

**Issue by ghost** [libretro/RetroArch#9367]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

On my Wii U with Mocha CFW and CBHC, I've installed the latest version of RetroArch with all cores and such that I need. However, I have all of my ROMs on a USB Drive and whenever I go to my USB Drive, RetroArch isn't picking it up and no folders or anything is showing. The USB Drive is FAT32 and is not formatted for Wii U games so that shouldn't be an issue. It is 250 gb and the same drive works with USB Loader GX on the vWii mode. I know that the Wii U is picking it up but not sure why RetroArch can't see it. I was really looking forward to playing some classics so I'm hoping this can get fixed!

### Expected behavior

RetroArch to load ROMs from my USB Drive

### Actual behavior

RetroArch doesn't see anything on my USB Drive

### Steps to reproduce the bug

1. Get Mocha CFW and CBHC
2. Launch RetroArch from Homebrew Launcher
3. Go to usb/ and for me, it doesn't show any games.

### Bisect Results

This started happening yesterday when I first installed it.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.8

### Environment information

- OS: Wii U
- Compiler: Wii U App Store

**Comment by jeremyshor** [comment]
I'm having the same problem with my 1GB SanDisk SSD Plus using the Black Wii U and loading just from Haxchii. I had formatted the drive to FAT32 using Chromea OS.

**Comment by geogolem** [comment]
Did you guys ever solve this problem. Similar issue here.

**Comment by NotAnemic** [comment]
I wish. No response from the team either.

**Comment by geogolem** [comment]
It seems you can now inject the retroarch roms into the WIIU usb instead of the fat32 USB you might use for vWII..

Fat32 is still nicer since you can manage the roms on the PC directly without having to use ftpIIU etc.

**Comment by NotAnemic** [comment]
Yeah, that's what I've been doing for mine in the mean time. Hopefully they can work out a fix eventually.

**Comment by filobeddo** [comment]
I'm having this one too.

I have a WiiU (CBHC/Haxchi) with a WiiU formatted drive (USB pen-drive) for WiiU titles. Then I have an USB hard-drive (Fat32/MBR*) for vWii/Gamecube and (I hope) ROMs for emulators (such as Retroarch).

The vWii has no problem seeing the Fat32 files.
When in WiiU/RetroArch I find that browsing to usb:/ yields no folder/file listing; sometimes!

Sometimes being that on occasion I HAVE seen my USB/Fat32 file-structure and can load a ROM with no issue.

What I note of interest is:

 * WHEN I CAN see my USB/FAT32 I have nothing listed under /storage_usb/usr
 * When I CAN’T see my USB/FAT32 then I CAN see /usr (and sub-folders) under /storage_usb

I am wondering if this intermittence indicates something about the order/numbering the WiiU is presenting the two drives? e.g. on some occasions it numbers the WiiU native is some lower number than the fat32? or vice/versa ?

Just a thought.

*My USB/FAT32 actually have four partitions (three of zero-size) as per the recommendation in this guide: https://gbatemp.net/threads/the-definitive-vwii-hacking-guide.425852/#14
